### ISSUE TITLE
Feral Goblins seem to crash the game sometimes.

### ISSUE THREAD

**Issue by SirAron111** [Silentine/GrimoireOfGaia#130]
## Client Info
* Minecraft Version: 1.12.2 Modpack Dimension Zero version 7.4

* Forge Version: forge-14.23.5.2825
* Mod Version: GrimoireOfGaia3-1.12.2-1.6.9.3
* Java Version: Java8 64x bit

Feral Goblin crashed the game for some reason

Don’t know it was just there and gave this crash normally have them spawn allot but they never crashed before.

https://pastebin.com/1wZKgwQp crash report

**Comment by SirAron111** [comment]
still sometimes crashing the game these little dudes https://pastebin.com/8E7cdFJ1

**Comment by SirAron111** [comment]
yep another user reported this happening its rarely but it does happen sometimes https://pastebin.com/huTRdpVD

**Comment by Silentine** [comment]
It seems to be a `ticking entity` issue, most likely caused by the Bomber type Feral Goblin. I always felt the code they used was a bit shaky. I'll assume you made them spawn naturally as they currently cannot spawn in any biome. Perhaps this may be causing the issue. Nevertheless, they should still work fine despite this.

Thanks for reporting this potential bug. I will definitely look into it once development resumes.

_(Sorry for my slow response. Development is currently on hiatus.)_

**Comment by SirAron111** [comment]
I spawn in the feral goblins with the rough mobs mod that can spawn mobs with the ID. The spawning works fine it’s a problem with them being bombers I think (feral goblins should just be weaponless goblins no bomber no bow or swords man would be better) keep the bomber for the normal goblins. (Thought this bug can also happen with normal goblin bombers)

**Comment by Silentine** [comment]
The Feral Goblins were added as a request for mod compatibility with another mod called **Ancient Warfare 2**. I can't remove them at this point.
The bug also happens with normal Goblin Bombers? Weird, the way normal Goblin Bombers work is identical to a lot of mobs in the mod that throw projectiles instead of shooting them.

**Comment by SirAron111** [comment]
Could be i remember it wrongly and it too was a feral goblin also you don’t need to remove them I meant remove the fact that they can spawn as bombers feral should stay feral no weapon filthy look compared to the normal goblins that look cool etc maybe only a club or spiked club but no bows and bombs these are to advanced for feral goblins XD

**Comment by Silentine** [comment]
Feral Goblin Bombers don't really use bombs normally. They use modified Creeper AI which is pretty new to the mod and the likely culprit for the crashes.
Feral Goblins in general were added to be weaker versions of the normal Goblins.

**Comment by Sereath** [comment]
I'm here to report the same issue.

https://gist.github.com/Sereath/ac0297f3ad76255485b5f4c8ee8f00b7

Spawned them by using InControl.
